app_dart/lib/src/service/luci.dart

```diff
@@ -194,8 +194,8 @@ class LuciBuilder {
   Map<String, dynamic> toJson() => _$LuciBuilderToJson(this);
 
   /// Loads and returns the list of known builders from the Cocoon [config].
-  static Future<List<LuciBuilder>> getBuilders(Config config) async {
-    final List<dynamic> builders = await config.luciProdBuilders;
+  static Future<List<LuciBuilder>> getBuilders(String repo, Config config) async {
```

Comment:
Should this be named as getProdBuilders or pass the bucket as parameter?

Reply:
Changed to `getProdBuilders`.